libs/fixtures/data/electionFamousNames2021/cdf-election.ts

```diff
@@ -0,0 +1,244 @@
+const cdfElection = {
+  '@type': 'BallotDefinition.BallotDefinition',
+
+  // Definition of the election (mainly candidates, contests, and ballot styles)
+  Election: [
+    {
+      '@type': 'BallotDefinition.Election',
+      ElectionScopeId: 'state-hamilton', // The GPUnit for this election
+      StartDate: '2023-02-01',
+      EndDate: '2023-02-01',
+      Type: 'general', // Other types: primary, runoff, special, etc.
+      Name: {
+        '@type': 'InternationalizedText',
+        Text: [
+          {
+            '@type': 'BallotDefinition.LanguageString',
+            Language: 'en',
+            Content: 'Lincoln Municipal General Election',
+          },
+        ], // Election name in other languages would go here
+      },
+
+      // Definition of all candidates in the election across contests
+      Candidate: [
+        {
+          '@type': 'BallotDefinition.Candidate',
+          '@id': 'sherlock-holmes',
+          BallotName: {
+            '@type': 'InternationalizedText',
+            Text: [
+              {
+                '@type': 'BallotDefinition.LanguageString',
+                Language: 'en',
+                Content: 'Sherlock Holmes',
+              },
+            ],
+          },
+          PartyId: 'party-0',
+        },
+        {
+          '@type': 'BallotDefinition.Candidate',
+          '@id': 'thomas-edison',
+          BallotName: {
+            '@type': 'InternationalizedText',
+            Text: [
+              {
+                '@type': 'BallotDefinition.LanguageString',
+                Language: 'en',
+                Content: 'Thomas Edison',
+              },
+            ],
+          },
+          PartyId: 'party-1',
+        },
+      ],
+
+      // Definition of contests
+      Contest: [
+        {
+          '@type': 'BallotDefinition.CandidateContest',
+          '@id': 'mayor',
+          ElectionDistrict: 'district-1',
+          // TODO what is the analogous field for "section"?
```

Comment:
I wasn't sure what the "section" field in the VX contest definition represented, so it wasn't clear how to represent that in CDF

Reply:
I think the way that we're handling `section` is not great. Despite the name, in the code all that `section` really means is a subtitle for the race. @mattroe, Beau, and I have done a lot of bike shedding about what we should use that subtitle for in our MS ballots. And you'll see in almost all cases in our fixtures, `section` is the same as the associated district name. But code and design-wise, it's just a subtitle. And unlike most subtitles, it goes _above_ the title for each contest.

I think we need some data on what titles and subtitles really need to go above races on ballots in states we want to enter. As far as creating a ballot from a CDFs, we could do a few things:
- Use `BallotSubTitle` if exists, have no subtitle otherwise
- Always use the name of the associated GpUnit
- Use `BallotSubTitle` if exists, the name of the GpUnit otherwise

As far as converting one of our definitions to CDF... I think it's hard because we haven't ever been clear on what exactly `section` is. But for now it could be BallotSubTitle or completely fine to drop IMO.

Reply:
It doesn't surprise me that there isn't an analogous field represented in the CDF because I haven't seen a field like our the intended use of our section on any other ballot. We don't even use the section in the way Beau intended when we convert from SEMS.

On all of our VotingWorks ballots in MS, which are converted from SEMS, the "section" on the ballot is most similar to the election district value and in many cases is duplicative of the district name in the election definition. As Beau explained it to me, this was not the original intention of this field - it was intended to be analogous to "level of government" such as "federal", "state", "local" etc. which is metadata uncommonly found on the ballots. I would be happy moving away from the section concept entirely and in our default ballot design aligning it with district as we do in MS today.

Reply:
I wrote my note before I saw @adghayes's comment, but overall I think we're on the same page. It's just a subtitle that we in practice use most commonly to display the "district"

Reply:
Great context. I like the idea of just using the GpUnit name if that's what we're using `section` for today. Easy to add support for `BallotSubTitle` later on.